src/main/java/net/glowstone/block/blocktype/BlockCactus.java

```diff
@@ -123,4 +127,17 @@ private boolean canPlaceNear(Material type) {
                 return true;
         }
     }
+
+    /**
+     * Get the items that will be dropped by digging the block.
+     * Specially overriten for cactus to remove data from the dropped item.
```

Comment:
Javadocs are not required for overridden methods, I think a comment saying `Overridden for cactus to remove data from the dropped item` within the method would be sufficient. Same for sugar cane.